### PR TITLE
Introduce AT-AT 660M

### PR DESCRIPTION
**Pull request by adrientetar**
Introduce AT-AT 660M keyboard PCB.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/at_at/660m/readme.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+AT-AT 660M
```
Comment (fauxpark) [inline keyboards/at_at/660m/readme.md]:
```suggestion
# AT-AT 660M
```

File: keyboards/at_at/660m/readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+==========
+
+660M keyboard
+
+Keyboard Maintainer: [Adrien Tétar](https://github.com/adrientetar)
```
Comment (fauxpark) [inline keyboards/at_at/660m/readme.md]:
```suggestion
Keyboard Maintainer: [Adrien Tétar](https://github.com/adrientetar)  
```

File: keyboards/at_at/660m/readme.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Hardware Supported: AT-AT 660M
+
+
+Make example for this keyboard (after setting up your build environment):
+
+    make at_at/660m:default
+
+See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
```
Comment (fauxpark) [inline keyboards/at_at/660m/readme.md]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

**Comment by mechmerlin** [comment]
Can you provide more information as to what this is? Your readme says it's a 660m keyboard, which already exists as a product from Leopold. Is this a replacement PCB for that?

**Comment by adrientetar** [comment]
> Is this a replacement PCB for that?

It fits the FC660M from Leopold with a slight dremeling of the case (as it's USB-C).

**Comment by adrientetar** [comment]
Done, PTAL

File: keyboards/at_at/660m/readme.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# AT-AT 660M
+
+660M keyboard
+
+Keyboard Maintainer: [Adrien Tétar](https://github.com/adrientetar)  
+Hardware Supported: AT-AT 660M
+
```
Comment (noroadsleft) [inline keyboards/at_at/660m/readme.md]: The Hardware Availability information (links to vendors or a repository if you're open-sourcing the hardware) should go here on Line 7 when you're ready.

Everything else looks good to me.

**Comment by noroadsleft** [comment]
Looks good, thanks!

**Comment by adrientetar** [comment]
Thanks. I'm trying QMK configurator, but it can't load the default keymap. What should I change to have the default keymap load?

**Comment by noroadsleft** [comment]
@adrientetar,

QMK Configurator's default keymaps live here: https://github.com/qmk/qmk_configurator/tree/master/public/keymaps

It basically takes a JSON version of the keyboard's `keymaps/default/keymap.c`.

If you fork qmk/qmk_configurator, you can then use the site itself to make a keymap, export the JSON to that folder in your local repo, push it back to GitHub, and then open a pull request.

I can actually build one for you and have it up probably in a few hours if you'd rather me do that. I do most of the maintenance for Configurator's default keymaps so I have a pretty good workflow for making them, but my workflow isn't actually on GitHub. (I actually do most of the work by hand.)

**Comment by adrientetar** [comment]
@noroadsleft If you don't mind making one from the [.c keymap](https://github.com/qmk/qmk_firmware/pull/6729/files#diff-f0d57814682ebab2a602a7b0c9840070), that's great! Otherwise I'll look into it when I have some time, thanks for the guidance!

**Pull request merged**